x-pack/winlogbeat/module/sysmon/config/winlogbeat-sysmon.js

```diff
@@ -506,22 +653,72 @@ var sysmon = (function () {
         .Add(removeEmptyEventData)
         .Build();
 
+    // Event ID 3 - Network connection detected.
     var event3 = new processor.Chain()
         .Add(parseUtcTime)
+        .AddFields({
```

Comment:
Some of these `add_fields` processors are missing targets so the fields are being written as `fields.event.category`. Maybe we should set `target: event` then use `fields: {category: ...`.

I deployed the latest snapshot and the data is our siem dev cluster if you want to see it.